osquery/tables/events/darwin/openbsm_events.cpp

```diff
@@ -156,10 +196,106 @@ Status OpenBSMExecVESubscriber::Callback(
     }
   }
   r["uptime"] = INTEGER(tables::getUptime());
+
+  try {
```

Comment:
I think I would prefer a check inclusion than an throw catch here.

@theopolis

Reply:
Use a find here, then you will have a pointer to the object or know it's not there.

Reply:
Addressed in the latest update.